{"name": "测试数据：【 admin，admin，True，0，登录成功】", "status": "broken", "statusDetails": {"message": "TypeError: login_user() takes 0 positional arguments but 2 were given", "trace": "self = <testcases.api_test.test_03_login.TestUserLogin object at 0x0000016B224D0CF8>, username = 'admin', password = 'admin', except_result = True, except_code = 0, except_msg = '登录成功'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n>       result = login_user(username, password)\nE       TypeError: login_user() takes 0 positional arguments but 2 were given\n\ntestcases\\api_test\\test_03_login.py:29: TypeError"}, "description": "该用例是针对获取用户登录接口的测试", "attachments": [{"name": "log", "source": "b19545fb-dc46-4db8-bcba-00600013c41f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'admin'"}, {"name": "password", "value": "'admin'"}, {"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "'登录成功'"}], "start": 1600161454385, "stop": 1600161454385, "uuid": "1f5c4dc2-6d9d-457d-9854-a2240f6d9621", "historyId": "b4ec1b9500ce5ee03ff0a85f4ade4115", "testCaseId": "957e450fb6ceaa933c3696671a75c4d9", "fullName": "project.pytestDemo-master.pytestDemo-master.testcases.api_test.test_03_login.TestUserLogin#test_login_user", "labels": [{"name": "feature", "value": "用户登录模块"}, {"name": "severity", "value": "normal"}, {"name": "story", "value": "用例--登录用户"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "project.pytestDemo-master.pytestDemo-master.testcases.api_test"}, {"name": "suite", "value": "test_03_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "DESKTOP-68E5LL7"}, {"name": "thread", "value": "8116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "project.pytestDemo-master.pytestDemo-master.testcases.api_test.test_03_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}